src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.UnitConverter/InputInterpreter.cs

```diff
@@ -1,7 +1,8 @@
-﻿// Copyright (c) Microsoft Corporation
+// Copyright (c) Microsoft Corporation
```

Comment:
I think you've added some weird characters at the beginning of the file. (Might be BOM) 
Could you please remove those characters?

Reply:
I checked both files in a hex editor:

![bom](https://user-images.githubusercontent.com/97686349/157510212-f7addf40-3031-4d09-a32d-b01d1b3591c4.png)
It turns out that the original file (right) has a BOM at the beginning while mine (left, using System.Collections.Generic) does not.
I have no idea why it suggests that I added a character when I actually deleted one without realizing it.

Could you tell me which would you suggest? Add a BOM to keep it the same as it was, or just delete the BOM?

Reply:
It was the other way around? Sounds good, then. Sorry for the confusion.